### DISCUSSION TITLE
Is the Rust SDK officially supported?

### DISCUSSION THREAD

**Discussion by destin-estrela** [deepgram/community#1013] [General help] [answered]
I'm seeing a conflict with the README.md and the description of the rust crate. The README.md says its community maintained, but the crate title says its official.

Could anyone clarify what the current state of the Rust SDK is, and if its stable and feature complete to begin developing off of? Thank you!

**Comment by deepgram-community** [comment]
Hey there! It looks like you haven't connected your GitHub account to your Deepgram account. You can do this at https://community.deepgram.com - being verified through this process will allow our team to help you in a much more streamlined fashion.

**Comment by deepgram-community** [comment]
It looks like we're missing some important information to help debug your issue. Would you mind providing us with the following details in a reply?

- The deepgram product you are using (e.g Speech to Text, Agent API)
- A request ID that triggered your error or issue.

**Reply by destin-estrela** [reply]
I'm interested primarily in the Speech to text product and there was no request ID.

**Comment by jpvajda** [comment, marked as answer]
@destin-estrela  the Rust SDK is currently consider a community SDK. we've corrected that on Crate.io so now it should be clearly communicating that status in our recent release.

https://crates.io/crates/deepgram
